mrp_mto_with_stock/models/mrp_production.py

```diff
@@ -122,28 +119,21 @@ def run_procurement(self, move, qty, mto_with_no_move_dest_id):
             raise UserError('\n'.join(errors))
         return True
 
-    @api.model
-    def _get_incoming_qty_waiting_validation(self, move):
-        """
-        This method should be overriden in submodule to manage cases where
-        we need to add quantities to the forecast quantity. Like draft
-        purchase order, purchase request, etc...
-        """
-        return 0.0
```

Comment:
why do you remove this hook?

Reply:
There is no more a submodule that needs this logic.
Anyway, any future module which would need this, can add this function.

Reply:
The module `mrp_mto_with_stock_purchase` was using it, and the idea of having this here is that you can extend that logic in different submoules and they would be able to coexist or be installed individually.

Reply:
The module `mrp_mto_with_stock_purchase` is no more needed and won't be migrated, if you think this is needed please explain why and migrate it.
Anyway this is dead code until someone add a module that uses it, and in that possible future PR that add this module, that code can be added too, without problem.

Reply:
Ok, fair enough